about/get-wpe.md

```diff
@@ -163,8 +163,15 @@ a good way to track down any bug you might find and to fix it.
 * [WPEBackend-fdo](https://github.com/Igalia/WPEBackend-fdo).
 * [Cog](https://github.com/Igalia/cog).
 
-Instead of downloading each of these components on their own, a good way to start is to fetch the WebKit repository
-and use its build system to fetch all the dependencies. Check the [instructions for building WebKit](https://trac.webkit.org/wiki/WPE#BuildingWPEWebKitfromgit).
+Instead of downloading each of these components on their own, the recommended
+procedure is cloning Git repositories for WebKit and the [WebKit Container
+SDK](https://github.com/Igalia/webkit-container-sdk). Follow the SDK
+instructions to prepare the container and open a shell inside it; then
+running `Tools/Scripts/build-webkit --wpe` in the WebKit source directory
+will produce a developer build&mdash;all the [other development
+tools](https://docs.webkit.org/Getting%20Started/ContributingCode.html) can
+be run directly in the container as well.
+
```

Comment:
Has the SDK been publicly announced yet?

Reply:
Not yet, but it's already the way to go and the idea is to announce it before the WEH. /cc @TingPing

Reply:
In case it's not clear: we would like to have the links updated in the website already up-to-date when making the announcement.

At any rate, the previous links were not particularly, and were pointing to the Trac wiki, which has not been in active duty for a good while.